src/main/markdown/doc/latest/polymer-tutorial/widgets-buildui.md

```diff
@@ -7,13 +7,13 @@ In this chapter we'll build a modern looking UI for the **TodoList** application
 
 1.  Create the **main screen** of the application.
 
-    We will create a [UiBinder](DevGuideUiBinder.html) screen composed of a java file  (`Main.java`) and its visual descriptor (`Main.ui.xml`).
+    We will create a [UiBinder](http://www.gwtproject.org/doc/latest/DevGuideUiBinder.html) screen composed of a java file  (`Main.java`) and its visual descriptor (`Main.ui.xml`).
```

Comment:
Rather use a relative path (`../DevGuideUiBinder.html`) than an absolute URI.

Reply:
Thanks for the feedback, I changed the absolute to a relative path.